qt/qt_common/map_widget.cpp

```diff
@@ -118,6 +126,22 @@ void MapWidget::ScalePlusLight() { m_framework.Scale(Framework::SCALE_MAG_LIGHT,
 
 void MapWidget::ScaleMinusLight() { m_framework.Scale(Framework::SCALE_MIN_LIGHT, true); }
 
+void MapWidget::MoveRight() { m_framework.Move(-0.2, 0, true); }
```

Comment:
Consider introducing two static constexpr constants for these values which state the units and the meaning.

Reply:
Thanks for the suggestion! I have done that. Let me know if you find the name of the variable appropriate (I could not find a better name)

Reply:
The DCO should also be fixed now.